metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -532,20 +536,40 @@ public void ingestEntity(@Nonnull Entity entity, @Nonnull AuditStamp auditStamp,
             .collect(Collectors.toList())));
   }
 
+  private boolean shouldHaveAspect(String entityType, String aspectName, Set<String> includedAspects) {
```

Comment:
Let's add a small comment above to note what this function does. Was a bit confused at first!